### PR TITLE
fix: Fixed E731 errors in tests

### PR DESCRIPTION
**Pull request by ashm-tech**
# Thanks for contributing!

Before submitting your pull request please have a look at the
following checklist:

- [x] ran the tests (`pytest`)
- [x] all style issues addressed (`flake8`)
- [x] your changes are covered by tests
- [x] your changes are documented, if needed

In addition, please take care to provide a proper description
on what your change does, fixes or achieves when submitting the 
pull request.

### CODE REVIEW

File: tests/test_format.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def test_strip_comments_preserves_linebreak(self):
         assert res == 'select *\n\nfrom foo'
 
     def test_strip_ws(self):
-        f = lambda sql: sqlparse.format(sql, strip_whitespace=True)
+        def f(sql): return sqlparse.format(sql, strip_whitespace=True)
```
Comment (andialbrecht) [inline tests/test_format.py:105]: ah, I've got a different opinion on this. E731 states to prefer a function definition over assigning a lambda expression to a variable. But as also stated in E731 the primary reason is for debugging as the lambda expression shows up as `<lambda>` in a traceback. 

But here the lambda is just a shorthand for the formatting code to make the tests shorter and more readable. Iff debugging is needed here, there's only one `<lambda>`. So it's pretty obvious. 

On the other hand I find single line function definitions a bit harder to read and I would avoid them if possible.

So to me "Readability counts" is the argument here, especially as the one reason for using function definitions instead of assigning a lambda function to a variable doesn't count here.

File: tests/test_format.py [hunk 2]
```diff
@@ -140,8 +140,7 @@ def test_notransform_of_quoted_crlf(self):
 
 class TestFormatReindentAligned:
     @staticmethod
-    def formatter(sql):
-        return sqlparse.format(sql, reindent_aligned=True)
+    def formatter(sql): return sqlparse.format(sql, reindent_aligned=True)
```
Comment (andialbrecht) [inline tests/test_format.py:143]: Why did you change this to a single line expression? It's harder to read, IMO.

File: tests/test_format.py [hunk 3]
```diff
@@ -616,26 +614,27 @@ def test_insert_values(self):
...
             "sql = 'select * from foo; '",
             "sql2 = 'select 1 from dual'"])
 
     @pytest.mark.xfail(reason="Needs fixing")
     def test_python_multiple_statements_with_formatting(self):
         sql = 'select * from foo; select 1 from dual'
-        f = lambda sql: sqlparse.format(sql, output_format='python',
-                                        reindent=True)
+        def f(sql): return sqlparse.format(sql, output_format='python', reindent=True)
+        from pprint import pprint
```
Comment (andialbrecht) [inline tests/test_format.py:636]: Looks like you've left some debugging code in this change.

File: tests/test_format.py [hunk 4]
```diff
@@ -616,26 +614,27 @@ def test_insert_values(self):
 class TestOutputFormat:
     def test_python(self):
         sql = 'select * from foo;'
-        f = lambda sql: sqlparse.format(sql, output_format='python')
+        def f(sql): return sqlparse.format(sql, output_format='python')
         assert f(sql) == "sql = 'select * from foo;'"
-        f = lambda sql: sqlparse.format(sql, output_format='python',
-                                        reindent=True)
+        def f(sql): return sqlparse.format(sql, output_format='python', reindent=True)
```
Comment (andialbrecht) [inline tests/test_format.py:619]: This and some of the following changes break the 80-column rule.

File: tests/test_format.py (hunk 1 above)
Comment (ashm-tech) [reply tests/test_format.py:105]: Okay, I get you!)
I will close this PR now,
and thank you for your feedback
